content/en/security/threats/workload_security_rules/_index.md

```diff
@@ -19,9 +19,68 @@ further_reading:
   text: "Learn more about Security notification variables"
 ---
 
-With Cloud Security Management Threats (CSM Threats) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of out-of-the-box rules to detect suspicious behavior. CSM Threats rules consist of two different components: [Agent rules](#agent-rules) and [detection rules](#detection-rules).
+This topic explains how Cloud Security Management Threats (CSM Threats) actively monitors system activity and evaluates it against a set of out-of-the-box (OOTB) rules to detect suspicious behavior.
 
-## Agent rules
+## CSM Threats rules construction
+
+CSM Threats rules consist of two different components: Agent rules and Threat detection rules.
+
+- **Agent rules:** [Agent rules][9] are evaluated on the Agent host. CSM Threats first evaluates activity within the Datadog Agent against Agent expressions to decide what activity to collect. Agent expressions use Datadog’s [Security Language (SECL)][2]. 
```

Comment:
```suggestion
- **Agent rules:** [Agent rules][9] are evaluated on the Agent host. CSM Threats first evaluates activity within the Datadog Agent against Agent expressions to decide what activity to collect. Agent expressions use Datadog's [Security Language (SECL)][2]. 
```

Reply:
Adding a suggestion to use two break (< br >) HTML elements so that there's a blank line between this block of text and the sentence that starts with "For example..."